cypress/support/index.js

```diff
@@ -0,0 +1,23 @@
+// ***********************************************************

```

Comment:
Not sure this file needs to be included

Reply:
you need this file to add extensions like cypress-axe.